__tests__/app.test.js

```diff
@@ -21,3 +27,28 @@ describe("CORE: GET /api/topics", () => {
       });
   });
 });
+
+describe("CORE: GET /api/articles", () => {
+  it("retrieves array of articles in decending order by date", () => {
```

Comment:
You are asserting that they are in the correct order in your description but have not tested for that.